.gitignore

```diff
@@ -167,3 +167,8 @@ tags
 
 # ruff
 .ruff_cache
+slurm-*.out
+run*.py
+run*.sh
+dataset.py
+prompts.py
```

Comment:
This should be removed, it might cause problems to future commits (unlikely, but possible :D)